packages/builder/src/package.ts

```diff
@@ -37,7 +37,7 @@ export interface PackagePublishCall {
 export class PackageReference {
   static DEFAULT_TAG = 'latest';
   static DEFAULT_PRESET = 'main';
-  static PACKAGE_REGEX = /^(?<name>@?[a-z0-9][A-Za-z0-9-]{1,}[a-z0-9])(?::(?<version>[^@]+))?(@(?<preset>[^\s]+))?$/;
+  static PACKAGE_REGEX = /^(?<name>[a-z0-9][A-Za-z0-9-]{1,}[a-z0-9])(?::(?<version>[^@]+))?(@(?<preset>[^\s]+))?$/;
```

Comment:
we should avoid breaking changes and remain backwards compatible probably, its ok to still have this in the regex

Reply:
See here
https://github.com/usecannon/cannon/pull/1198/files#diff-ddd0df427184818e574abaf99eb65fa94549dc69884461cc1b32f5236f1dadccR507

Reply:
Does it also get normalized in the web UI? Does it need to be normalized in the web UI? I don't actually know

Reply:
Ive changed the code on the website that uses '@ipfs://' to use `ipfs://` and it since the website uses the builder `getUrl` function and thats been changed to normalize `@ipfs` urls to `ipfs://` it shouldnt need any more changes on the website.